docs/Changelog.md

```diff
@@ -14445,8 +14445,9 @@ This version of the operator has been available since version 12 of the default
 
   Calculates inverse of a square matrix or batches of square matrices.
   Inverse takes one input tensor of shape `[*, M, M]`, where `*` is zero or more batch dimensions,
-  and the inner-most 2 dimensions form square matrices.
-  The output is a tensor of shape `[*, M, M]`, containing the individual inverses of all input submatrices.
+  and the inner-most 2 dimensions form square matrices. The matrices must be invertible (have full rank).
+  The behavior is undefined otherwise. The output is a tensor of shape `[*, M, M]`, containing the individual
```

Comment:
Would it make sense to say more here? Some something like: `The matrices must be invertible (full-rank). The behavior where one of the matrices is not invertible is undefined. The implementation can choose to throw an error or output (garbage) results as is.`
@gramalingam @neginraoof @yuslepukhin